code/_helpers/turfs.dm

```diff
@@ -112,6 +112,11 @@
 		return TRUE
 	return FALSE
 
+/proc/is_dark(turf/T, darkness_threshold = 0.3)
+	if(T.get_lumcount() <= 0.3)
```

Comment:
```suggestion
	if(T.get_lumcount() <= darkness_threshold)
```
Pretty sure you meant to do this, judging by this particular unused proc argument.

Reply:
Lmao I cant believe I missed that